owncloudApp/src/main/java/com/owncloud/android/presentation/sharing/shares/PublicShareDialogFragment.kt

```diff
@@ -321,6 +343,70 @@ class PublicShareDialogFragment : DialogFragment() {
         })
     }
 
+    private fun updateRequirements(password: String, textView: TextView) {
+        val hasMaxCharacters = password.length <= 72
+        val hasMinCharacters = password.length >= 8
+        val hasUpperCase = password.any { it.isUpperCase() }
+        val hasLowerCase = password.any { it.isLowerCase() }
+        val hasSpecialCharacter = password.any { SPECIALS_CHARACTERS.contains(it) }
+        val hasDigit = password.any { it.isDigit() }
```

Comment:
This is the main point of this issue, and it is not ok. You have to fetch the corresponding values from the capabilities!! Otherwise, why would have we done all the work fetching them from the server and saving them in database? 😰